rxandroidble/src/main/java/com/polidea/rxandroidble/RxBleClientImpl.java

```diff
@@ -66,6 +66,17 @@ public RxBleDevice getBleDevice(@NonNull String macAddress) {
     }
 
     @Override
+    public Set<RxBleDevice> getBondedDevices() {
+        Set<RxBleDevice> rxBleDevices = null;
```

Comment:
I believe this line should look: `Set<RxBleDevice> rxBleDevices = new HashSet(size);`
Otherwise the function will throw NPE in line 73 or I have missed where the `rxBleDevices` is set.